src/__tests__/ParserComponent.test.tsx

```diff
@@ -352,18 +352,26 @@ describe('<ParserComponent /> Unit Tests', () => {
 
     it('handles regular data', () => {
       const t1 = pc.inferTypes(data);
-      // test string
-      expect(t1[2].drawType).toBe(enumDrawType.occurrence);
-      // test number
-      expect(t1[0].drawType).toBe(enumDrawType.any);
-      // test date
-      // expect(t1[1].drawType).toBe(enumDrawType.any);
+      if (t1 !== undefined) {
+        // test string
+        expect(t1[2].drawType).toBe(enumDrawType.occurrence);
+        // test number
+        expect(t1[0].drawType).toBe(enumDrawType.any);
+        // test date
+        // expect(t1[1].drawType).toBe(enumDrawType.any);
```

Comment:
Why is this commented out?

Reply:
removed the comment and it passed test so line 361 is not a comment now